### ISSUE TITLE
[Design] On-site Notifications: Work with Content on notifications on va.gov

### ISSUE THREAD

**Issue by ajakabcin** [department-of-veterans-affairs/va.gov-team#31615]
## Background
Once we're through the design intent, we should start working with content on getting things reviewed and approved.

## Tasks
- [x] Submit content tickets as needed for review
- [x] Update designs based on content feedback
- [ ] Create/update [copy documentation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/notifications/vagov-notifications/mvp/frontend/documentation/notifications-copy-documentation.md)

**Comment by hdjustice** [comment]
Moved ticket to next sprint (11/3-11/16) per request from Tressa.

**Comment by andaleliz** [comment]
@tressaellen If I understand the new direction of onsite notifications correctly (no dedicated page, only the one payment alert), the new design that will need to be reviewed is shown in your [Round 4 payment mock-ups](https://app.mural.co/t/vsa8243/m/vsa8243/1638500075560/0e849f61e282ebe2d8e7119eea504e4c05b0bbe4?wid=0-1639773433074), specifically the alert for a new debt being added at the top of page, and the verbiage for the outstanding debts in the payment section.

Is that correct? Just want to try to get this all addressed in office hours this week. Thanks!

**Comment by andaleliz** [comment]
Chatted with Tressa and she confirmed I am correct, with the addition of the fact that we need to review the lack of a heading for the notification section.

**Comment by tressaellen** [comment]
Here's the [latest mural board](https://app.mural.co/t/vsa8243/m/vsa8243/1640028268317/2ba4e1b2846c87f5d0bfc6bcb643bef84720bea1?sender=tressafurner7646) with the latest design for onsite notifications and payment information! Thanks!

**Comment by andaleliz** [comment]
@hdjustice FYI I'm going to review this in office hours tomorrow so I've moved this into the current sprint.

**Comment by hdjustice** [comment]
@andaleliz Got it, thank you!

**Comment by andaleliz** [comment]
I reviewed this with Mikki and Danielle today in office hours. They had one small tweak to the veribage, which has been updated in [the Sketch files](https://www.sketch.com/s/418ca6f6-cc24-4496-82aa-9deda03b08f6). I haven't updated the Mural board.

You have new debt. [Manage your VA debt](link to debt page)

It's longer, but linking the words "you have 3 outstanding debts" is not clear as to where the link will take the user. Their suggestion is also consistent with how this is handled in other parts of the site.

I asked whether or not we need a heading for this section, and they didn't feel strongly one way or the other. I believe @joshkimux advised that for a11y purposes a section heading isn't necessary, so for now, we can leave it out.

Mikki and Danielle both feel that it's possible we will need a separate treatment for onsite notifications vs the notifications within each section; they commented that it feels odd that the one is visually and content-wise basically identical to the others, but it's behavior is slightly different. I agree with them, but none of us feel strongly one way or another and decided we should let research dictate our next move here.
